torch/_tensor_str.py

```diff
@@ -206,7 +214,7 @@ def _vector_str(self, indent, summarize, formatter1, formatter2=None):
     elements_per_line = max(
         1, int(math.floor((PRINT_OPTS.linewidth - indent) / (element_length)))
     )
-    char_per_line = element_length * elements_per_line
+    # char_per_line = element_length * elements_per_line  # unused
```

Comment:
just delete it then lol (don't do it here, can do in follow up)

Reply:
I get nervous when I see something obvious like this in someone else's code. Is it really unused? Maybe it causes some side effect I'm not aware of? 

In this case I was convinced enough to comment it out, but unsure of myself enough to not delete it outright. But I'll remove it in the follow-up.